pulseapi/entries/models.py

```diff
@@ -174,7 +174,7 @@ class Meta:
         Make plural not be wrong
         """
         verbose_name_plural = "entries"
-        ordering = ['id']
+        ordering = ['-id']
```

Comment:
let's not do this at the model level coz it forces `order by` clauses on querysets that don't need it

Reply:
consequently, add the order_by clause for `published_entries` as well

Reply:
It's already forcing the clause, I'm just changing the default. Besides, the `published_entries` aren't fetched the same way as `created_entries` so I can't seem to simply stick on `ordered_by` anywhere obvious.

Reply:
ok, if we're sure we want reverse chronological ordering for every type of fetch on a list of entries then this is fine. Otherwise, `published_entries` are fetched here: https://github.com/mozilla/network-pulse-api/pull/333/files#diff-9caaebc8dbfd9b041d5714a6e437bd47R148, you just need to chain the `order_by` query there.